### PR TITLE
add snapshot option 

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Add SnapShot option for meshViewer and 3dVolViewer  (useful to get visualisation without interaction like for scripting and/or online demonstration)

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

**Comment by kerautret** [comment]
can I merge ?

File: visualisation/3dVolViewer.cpp [hunk 1]
```diff
@@ -78,6 +79,10 @@ using namespace Z3i;
                                      Hounsfield scale
   --dicomMax arg (=3000)             set maximum density threshold on 
                                      Hounsfield scale
+  --displayMesh arg                display a Mesh given in OFF or OFS format. 
+  --colorMesh arg                  set the color of Mesh (given from 
+                                   displayMesh option) : r g b a 
+  -d [ --doSnapShotAndExit]  arg,  save display snapshot into file. Notes that the camera setting is set by default according the last saved configuration (use SHIFT+Key_M to save current camera setting in the Viewer3D). If the camera setting was not saved it will use the default camera setting.
```
Comment (dcoeurjo) [inline visualisation/3dVolViewer.cpp]: Replace arg by filename ?

Comment (kerautret) [inline visualisation/3dVolViewer.cpp]: yes I change it thanks ;)

**Comment by kerautret** [comment]
done

**Comment by dcoeurjo** [comment]
thanks merging.

**Pull request merged**